coldfront/templates/common/authorized_navbar.html

```diff
@@ -35,7 +35,7 @@
           <a class="nav-link" href="{% url 'qumulo:allocation' %}">Create Allocation</a>
         </li>
         <li id="navbar-view-allocation" class="nav-item">
-          <a class="nav-link" href="{% url 'qumulo:allocation-table-list' %}">View All Allocation</a>
+          <a class="nav-link" href="{% url 'qumulo:allocation-table-list' %}">View All Allocations</a>
```

Comment:
@ris-bmulligan thanks for fixing the spelling. A minor change that makes the app presentable.